### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by ritz301**

### CODE REVIEW

File: popup.html [hunk 1]
```diff
@@ -1,234 +1,20 @@
...
-    return [true, ""];
-  }
-});
-$('#datetimepicker_dark').datetimepicker({theme:'dark'})
-
-
-</script>
-</html>
+<script src="./popup.js"></script>
+<html>
```
Comment (kshitij10496) [inline popup.html]: Add a newline at the end

File: popup.html [hunk 2]
```diff
@@ -1,234 +1,20 @@
...
-    <option value="es">Spanish</option>
-  </select>
-  <input type="text" value="D, l, M, F, Y-m-d H:i:s" id="datetimepicker_format_value"/>
-  <input type="button" value="applay =>" id="datetimepicker_format_change"/>
-  <input type="text" id="datetimepicker_format" class="input input-wide"/>
-  -->
-  <label for="note">Note about site: <span id="sitename"></span></label>
+	<h3>DateTimePicker</h3>
+	<input type="text" value="" id="datetimepicker"/><br><br>
+	<label for="note">Note about site: <span id="sitename"></span></label>
```
Comment (kshitij10496) [inline popup.html:12]: Cool work !

File: jquery.datetimepicker.css [hunk 3]
```diff
@@ -0,0 +1,568 @@
...
...9zaG9wIENTNiAoV2luZG93cykiIHhtcE1NOkluc3RhbmNlSUQ9InhtcC5paWQ6Q0NBRjI1NjM0M0UwMTFFNDk4NkFGMzJFQkQzQjEwRUIiIHhtcE1NOkRvY3VtZW50SUQ9InhtcC5kaWQ6Q0NBRjI1NjQ0M0UwMTFFNDk4NkFGMzJFQkQzQjEwRUIiPiA8eG1wTU06RGVyaXZlZEZyb20gc3RSZWY6aW5zdGFuY2VJRD0ieG1wLmlpZDpDQ0FGMjU2MTQzRTAxMUU0OTg2QUYzMkVCRDNCMTBFQiIgc3RSZWY6ZG9jdW1lbnRJRD0ieG1wLmRpZDpDQ0FGMjU2MjQzRTAxMUU0OTg2QUYzMkVCRDNCMTBFQiIvPiA8L3JkZjpEZXNjcmlwdGlvbj4gPC9yZGY6UkRGPiA8L3g6eG1wbWV0YT4gPD94cGFja2V0IGVuZD0iciI/PoNEP54AAAIOSURBVHja7Jq9TsMwEMcxrZD4WpBYeKUCe+kTMCACHZh4BFfHO/AAIHZGFhYkBBsSEqxsLCAgXKhbXYOTxh9pfJVP+qutnZ5s/5Lz2Y5I03QhWji2GIcgAokWgfCxNvcOCCGKqiSqhUp0laHOne05vdEyGMfkdxJDVjgwDlEQgYQBgx+ULJaWSXXS6r/ER5FBVR8VfGftTKcITNs+a1XpcFoExREIDF14AVIFxgQUS+h520cdud6wNkC0UBw6BCO/HoCYwBhD8QCkQ/x1mwDyD4plh4D6DDV0TAGyo4HcawLIBBSLDkHeH0Mg2yVP3l4TQMZQDDsEOl/MgHQqhMNuE0D+oBh0CIr8MAKyazBH9WyBuKxDWgbXfjNf32TZ1KWm/Ap1oSk/R53UtQ5xTh3LUlMmT8gt6g51Q9p+SobxgJQ/qmsfZhWywGFSl0yBjCLJCMgXail3b7+rumdVJ2YRss4cN+r6qAHDkPWjPjdJCF4n9RmAD/V9A/Wp4NQassDjwlB6XBiCxcJQWmZZb8THFilfy/lfrTvLghq2TqTHrRMTKNJ0sIhdo15RT+RpyWwFdY96UZ/LdQKBGjcXpcc1AlSFEfLmouD+1knuxBDUVrvOBmoOC/rEcN7OQxKVeJTCiAdUzUJhA2Oez9QTkp72OTVcxDcXY8iKNkxGAJXmJCOQwOa6dhyXsOa6XwEGAKdeb5ET3rQdAAAAAElFTkSuQmCC);
```
Comment (kshitij10496) [inline jquery.datetimepicker.css:114]: What is this ?
Isnt this a garbage value ?

File: popup.js [hunk 4]
```diff
@@ -1,150 +1,8 @@
...
-$('#datetimepicker12').datetimepicker({
-  beforeShowDay: function(date) {
-    if (date.getMonth() == dateToDisable.getMonth() && date.getDate() == dateToDisable.getDate()) {
-      return [true, "custom-date-style"];
-    }
-
-    return [true, ""];
-  }
+disabledDates:['1986/01/08','1986/01/09','1986/01/10'],
+startDate:  '1986/01/05'
```
Comment (kshitij10496) [inline popup.js]: The `startdate` should be today's date !

Remove `disabledDates`.
Dont fiddle with the commented out code.
Read them and understand their relevance

File: popup.js [hunk 5]
```diff
@@ -1,150 +1,8 @@
...
 });
-$('#datetimepicker_dark').datetimepicker({theme:'dark'})
-
-
-</script>
\ No newline at end of file
+$('#datetimepicker').datetimepicker({value:'2015/04/15 05:03',step:10});
```
Comment (kshitij10496) [inline popup.js]: Newline character

**Comment by kshitij10496** [comment]
squash the 3 commits into a single commit

File: jquery.datetimepicker.css (hunk 3 above)
Comment (ritz301) [inline jquery.datetimepicker.css:114]: No this file is a part of the plugin. Its needed later for design purposes.

Comment (kshitij10496) [inline jquery.datetimepicker.css:114]: I get that !
I was talking about the `url` of `background-image`

**Comment by kshitij10496** [comment]
What happened dude ?

**Pull request closed without merging**